### PR TITLE
ai: more meaningful variable names

### PR DESCRIPTION
**Pull request by potatoeggy**
Readability is a little low rn

### CODE REVIEW

**Review by vincentjguo — APPROVED** [review]
Thanks for the reformat, everything looks good

**Comment by potatoeggy** [comment]
you know what i think i'll wait until you merge with master

merge conflicts are not fun

**Comment by vincentjguo** [comment]
true i have no idea how to resolve conflicts

**Pull request closed without merging**
